src/app/profile-page/profile-page-security-form/profile-page-security-form.component.ts

```diff
@@ -123,13 +129,13 @@ export class ProfilePageSecurityFormComponent implements OnInit {
   }
 
   /**
-   * Check if the password is at least 6 characters long
+   * Check if the password is at least 8 characters long
    * @param group The FormGroup to validate
    */
   checkPasswordLength(group: FormGroup) {
     const pass = group.get('password').value;
 
-    return isEmpty(pass) || pass.length >= 6 ? null : {notLongEnough: true};
+    return isEmpty(pass) || pass.length >= 8 ? null : {notLongEnough: true};
```

Comment:
I thought we had agreed to remove the minimum length validation (see discussion above)?  This part still seems very confusing in terms of usability.  The UI now tells you your password "should be at least eight characters long."... but then if you enter a 8 character password, it will still reject that password if it doesn't meet the regex requirements defined on the backend.

If we cannot determine if a password is "valid" until it is submitted to the backend, then I think we need to *remove* this password length check from the frontend (the only checks we can have on the frontend would be that the passwords match & are non-empty).  

Otherwise, if a site chooses to change the backend regex to require a *12 character* password, then they will be forced to also update the hardcoded 8 character check in this UI (along with all the i18n keys that say "8 characters"